bot/cogs/fun.py

```diff
@@ -98,7 +98,7 @@ def __init__(self, bot: Bot):
             await ctx.send(f"Unknown emojis: {emoji_string}")
 
     @command()
-    async def xkcd(self, ctx: Context, number=None):
+    async def xkcd(self, ctx: Context, number: str = None):
```

Comment:
Why is a number a string? Also kwargs shouldn't have spaces around the `=` if i recall correctly.